awx/main/models/schedules.py

```diff
@@ -88,6 +88,15 @@ class Meta:
     def get_zoneinfo(self):
         return sorted(get_zonefile_instance().zones)
 
+    @classmethod
+    def get_zoneinfo_with_links(self):
+        zone_instance = get_zonefile_instance()
+        return_val = {'zones': sorted(zone_instance.zones), 'links': {}}
```

Comment:
I second Alan's comment.  This data structure seems awkward compared to, say, a list of objects like `{'zone': zone, 'link': link}`.